docs/howto/set-up-landscape-server-in-wsl.md

```diff
@@ -0,0 +1,83 @@
+# How to install Landscape server in a WSL instance
+
+While a Landscape server typically runs on external computers, here it will be
+set up on a WSL instance on a Windows machine.
```

Comment:
If we'd preserve this piece I think it'd be good to present a motivation for doing so, as hosting Landscape on a WSL instance is not a supported way per se. In my view, the motivation is to ease development and testing of UP4W for those who don't have another machine to host Landscape and find setting up VMs harder than trying to run it on WSL, thus I'd place this under `dev docs`

Reply:
I think that's a great idea @CarlosNihelton and this should definitely be considered in a future revision.

For this PR I just wanted to make sure the useful information was captured somewhere so that it did not get lost. It was an "extra" after the material had been removed from the main tutorial.

I will note your suggestions down for now. Providing a more explicit motivation should be done wherever this guide is ultimately located.

Whether it should be in `dev docs` is something I/we need to think about. This depends to an extent on what we think about the state of those dev docs, their organisation as a whole and where they will fit in the WSL docs.